projects/npm-tools/packages/npm-scripts/test/scripts/theme.js

```diff
@@ -7,30 +7,29 @@ const os = require('os');
 const path = require('path');
 
 const spawnSync = require('../../src/utils/spawnSync');
+const getFixturePath = require('../../support/getFixturePath');
 
 jest.mock('../../src/utils/spawnSync');
 
 // Use path.normalize to make tests behave uniformly on Linux and Windows.
 
 const {normalize} = path;
-const join = (...segments) => normalize(path.join(...segments));
-const resolve = (...segments) => normalize(path.resolve(...segments));
-
-const FIXTURES = resolve(__dirname, '../../__fixtures__/scripts/theme');
-const MODULES = join(FIXTURES, 'modules');
-const APPS = join(MODULES, 'apps');
-const NODE_MODULES = join(MODULES, 'node_modules');
-const FJORD = join(APPS, 'frontend-theme-fjord/frontend-theme-fjord');
-const CLASSIC = join(APPS, 'frontend-theme/frontend-theme-classic');
-const BAD = join(FIXTURES, 'not/the/modules/you/are/looking/for');
-const FRONTEND = join(APPS, 'frontend-theme');
-const STYLED = join(
+
+const FIXTURES = getFixturePath('scripts', 'theme');
+const MODULES = path.join(FIXTURES, 'modules');
+const APPS = path.join(MODULES, 'apps');
+const NODE_MODULES = path.join(MODULES, 'node_modules');
+const FJORD = path.join(APPS, 'frontend-theme-fjord', 'frontend-theme-fjord');
+const CLASSIC = path.join(APPS, 'frontend-theme', 'frontend-theme-classic');
+const BAD = path.join(FIXTURES, 'not', 'the', 'modules', 'you', 'are', 'looking', 'for');
+const FRONTEND = path.join(APPS, 'frontend-theme');
+const STYLED = path.join(
```

Comment:
I hope these still work. I don't remember exactly why, but that comment about "Use path.normalize to make tests behave uniformly on Linux and Windows" makes me think I had to add to use `normalize` here for some reason. 🤷

Reply:
We are going to see it when CI finishes :sweat_smile: . The theory is that it should work, but with paths you never know...

When I saw the code I strongly empathized with the person behind it :-D. It reminded me of my adventures with path in the bundler.